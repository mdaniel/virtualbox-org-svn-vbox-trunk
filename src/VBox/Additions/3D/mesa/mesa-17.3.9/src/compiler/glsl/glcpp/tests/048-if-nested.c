success_1
#if 0
failure_1
#if 1
failure_2
#else
failure_3
#endif
failure_4
#endif
success_2
