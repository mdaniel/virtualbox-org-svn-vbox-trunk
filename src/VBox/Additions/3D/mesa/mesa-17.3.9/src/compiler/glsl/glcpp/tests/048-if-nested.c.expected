success_1









success_2
